lib/statistics.ex

```diff
@@ -321,4 +321,60 @@ defmodule Numerix.Statistics do
   defp do_quantile(xs, h, hf) do
     Enum.at(xs, hf - 1) + (h - hf) * (Enum.at(xs, hf) - Enum.at(xs, hf - 1))
   end
+
+  @doc """
+  Calculates rolling mean, prepending zeros for the values that don't
+  complete the minimum count of elements.
+  ## Examples
+
+    iex> Statistics.rolling_mean([2.8440000000000003, 3.096, 3.672, 4.572, 4.284], 2)
+    [0.0, 2.97, 3.3840000000000003, 4.122, 4.428]
+  """
+  def rolling_mean(collection, count) when is_list(collection) and is_integer(count) do
```

Comment:
What happens when `count < 1`?

Reply:
Can you also please add typespecs for both functions?

Reply:
For consistency, please use `Tensor` instead of plain old lists. You can look at other functions for reference.